src/benchInsert.c

```diff
@@ -1923,6 +1923,13 @@ int insertTestProcess() {
             if (database[i].superTbls[j].insertRows == 0) {
                 continue;
             }
+            if (database[i].superTbls[j].non_stop) {
```

Comment:
there is a prompt() for such a requirement. it can skip the prompt if the user input -y. it helps the auto-test process ongoing.